src/index.ts

```diff
@@ -109,6 +109,12 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "Path to a current working directory",
             requiresArg: true,
         })
+        .option("local-variables-file", {
+            type: "string",
+            description: "Path to the project file variables",
+            requiresArg: false,
+            default: Argv.default.localVariablesFile,
```

Comment:
introducing this new pattern, so that `--help` will show the default values which i find quite useful..

![image](https://github.com/user-attachments/assets/8ec12df1-b846-45e6-a2aa-592167d1c3e2)

if we're ok with this, will open a new mr to introduce same pattern for the other `options`